app/views/info/_per_page_metrics.erb

```diff
@@ -0,0 +1,27 @@
+<% if @per_page_metrics %>
+<% per_page_metrics %>
+  <div id="per-page-metrics" class="metrics-panel">
+    <h2>Metrics for individual pages</h2>
+
+    <table>
+      <thead>
+        <tr>
+          <th class='page'>Page</th>
+          <th class='unique_pageviews'>Unique pageviews</th>
+          <th class='searches'>Searches started from this page</th>
+          <th class='problem_reports'>Problem reports</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% per_page_metrics.each do |t, a| %>
+          <tr>
+          <td class='page'><a href="<%= t %>"><%= t %></a></td>
```

Comment:
it'd be nice to have the page title here as the link text, rather than the URL itself